### PR TITLE
DFPL-1475: Duplicate notifications being sent to CTSC

### PR DESCRIPTION
**Pull request by AlistairEdwinOsborne**
https://tools.hmcts.net/jira/browse/DFPL-1475

Setup handle submitted callback to use new concurrency standard for postSubmitCallback
AfterSubmissionCaseDataUpdatedEventHandler to be async
Shift handle post submitted publish event to submitted

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Review by jamesrferguson1 — DISMISSED** [review]

File: service/build.gradle [hunk 1]
```diff
@@ -54,51 +54,58 @@ sourceSets {
...
 }
 
+/*tasks.withType(Test) {
+  useJUnitPlatform()
+
+  testLogging {
+    exceptionFormat = 'full'
+  }
+}*/
+
```
Comment (DanCatchpole) [inline service/build.gradle:105]: Any reason why we're keeping this commented out? Might be useful in future if we do get an exception, saves uncommenting + rerunning the build

File: service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/RespondentControllerSubmittedTest.java [hunk 2]
```diff
@@ -364,34 +387,39 @@ void shouldUpdateRepresentativesAccess() {
...
+
         // setup change field, submit to aac (org1), setup change field, submit to aac (org2), update case-summary
-        verify(concurrencyHelper, times(5)).submitEvent(any(),
-            eq(caseData.getId()), caseCaptor.capture()
-        );
+        checkThat(() -> {
+            verify(concurrencyHelper, times(5)).submitEvent(startEventCaptor.capture(),
+                eq(caseData.getId()), caseCaptor.capture()
+            );
+        }, Duration.ofSeconds(10));
```
Comment (DanCatchpole) [inline service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/RespondentControllerSubmittedTest.java:401]: Was there any issues with the checkThat in combination with the times(5) thing, I think we chained them before right, `times(5).timeout(10000)` ?

Comment (AlistairEdwinOsborne) [reply service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/RespondentControllerSubmittedTest.java:401]: yeah for this instance the standard times(5) one failed so swapped it back to the checkThat()

File: service/build.gradle (hunk 1 above)
Comment (AlistairEdwinOsborne) [reply service/build.gradle:105]: When this is in it fails the build so thought it would be good to leave it in so anyone who wants to can uncomment it

**Review by DanCatchpole — APPROVED** [review]

**Pull request merged**